registry/auth/token/accesscontroller.go

```diff
@@ -238,6 +238,11 @@ func checkOptions(options map[string]interface{}) (tokenAccessOptions, error) {
 	return opts, nil
 }
 
+var (
+	rootCertificateFetcher func(string) ([]*x509.Certificate, error) = getRootCerts
```

Comment:
nit: can we rename this to `rootCertFetcher`? It's shorter and it also nicely aligns with the `getRootCert` func assigned to it. Sorry, I should have mentioned this on the first review 😇

Reply:
np, renaming it in a bit